amica/__tests__/helpers/PhysicsTestHelpers.ts

```
@@ -0,0 +1,499 @@
+    if (bodyConfig.rotation) {
+      bodyDesc.setRotation(bodyConfig.rotation);
+    }
+
+    const body = this.world.createRigidBody(bodyDesc);
+    this.bodies.push(body);
+
+    // Create collider
+    let colliderDesc;
+    switch (colliderConfig.shape) {
+      case 'box':
+        const size = colliderConfig.size || { x: 0.5, y: 0.5, z: 0.5 };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)